app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -13,9 +13,12 @@ import AddStoryButton from '../story/AddStoryButton';
 import * as Story from 'libs/beta/constants';
 import PropTypes from 'prop-types';
 import { storyPropTypesShape } from '../../models/beta/story';
-import { projectBoardPropTypesShape } from '../../models/beta/projectBoard';
+import { projectBoardPropTypesShape, hasSearch } from '../../models/beta/projectBoard';
 import Notifications from '../Notifications';
 import { removeNotification } from '../../actions/notifications';
+import StorySearch from '../stories/StorySearch';
+import Search from './../stories/Search';
+import PortalSearchInput from './../../portals/PortalSearchInput';
```

Comment:
I don't think I like the idea of creating a `portals` folder, this name doesn't add anything to the semantics beyond what the name of the component already does and in the end, it's still a component. What do you think about keeping this component name but moving it to `components/search`? I think the component `StorySearch` should be moved there too. It's more related to the search than with the stories since it doesn't belong to any part of a story component.